### ISSUE TITLE
Advanced Search dropdown missing descriptive text label

### ISSUE THREAD

**Issue by posixeleni** [samvera/hyku#3051]
vpat.hykudemo.org
Version 7.1

### Descriptive summary

Add a descriptive text label on the Advanced search page dropdown form selection for "Find items that match [All/ANY] of". According to WCAG 2.2 4.1.2 Every form element should have a descriptive text label. To be accessible, that label needs to be associated with the form element in the HTML. This helps visitors using screen readers to understand what information is required, while also making it easier for speech-input users to control the form element.

See also:
WCAG 2.2: [Opens in new tab:Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)

### Rationale

Accessibility issue remediation

### Steps to reproduce the behavior

Go to: https://vpat.hykudemo.org/advanced
Click on SiteImprove Plugin and see issue shown in screenshot

<img width="1122" height="1050" alt="Image" src="https://github.com/user-attachments/assets/01d93e3a-7ef9-4d70-8679-c1afcca2390e" />

This is the HTML code that is the issue:

 ```
<div class="input-criteria">

      <div class="query-criteria">
        <h3 class="query-criteria-heading">
          Find items that match <select name="op" id="op" class="input-small"><option value="AND">all</option>
<option value="OR">any</option></select> of 
        </h3>
```
Found on all of these pages:

https://vpat.hykudemo.org/advanced
https://vpat.hykudemo.org/advanced?locale=de
https://vpat.hykudemo.org/advanced?locale=en
https://vpat.hykudemo.org/advanced?locale=es
https://vpat.hykudemo.org/advanced?locale=fr
https://vpat.hykudemo.org/advanced?locale=it
https://vpat.hykudemo.org/advanced?locale=pt-BR
https://vpat.hykudemo.org/advanced?locale=zh

### Related work
